src/realsense2-all.cmake

```diff
@@ -0,0 +1,119 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+if( NOT BUILD_SHARED_LIBS )
```

Comment:
Where do we condition this to windows only?

Reply:
We do not: it's for Linux, too.